### PR TITLE
Childcare updates draft

### LINKED ISSUE #153: Childcare info
Per the DjangCon organizers meeting today and update from @davidfischer, childcare info will need to be updated on the website as it becomes available. Perhaps this should go in the venue page under accessibility. https://2018.djangocon.us/venue/#accessibility

### PR DESCRIPTION
**Pull request by davidfischer**
Closes #153

Changes proposed in this PR:

- Add a new childcare blog with information on the childcare provider
- Add a link to the post on the venue page

## Questions

- How much of the childcare will be sponsored?
- Should we thank the sponsors who are sponsoring childcare?
- How will people get reimbursed? Email?
- Are we only sponsoring with Destination Sitters or can people use a different childcare provider?

### CODE REVIEW

**Comment by rebkin05** [comment]
@davidfischer thanks for the update! The text looks good, except for a couple small things:
- every mention of DjangoCon should be DjangoCon US
- "far away" on line 12 should be "faraway"

**Comment by KatherineMichel** [comment]
I don't have answers to those questions. Otherwise, the post lgtm.

**Comment by craigbruce** [comment]
> How much of the childcare will be sponsored?

Philly was $85/child/day (don't recall if we had any in Spokane)? Happy to match that if that is reflective of SD rates.

> Should we thank the sponsors who are sponsoring childcare?

Absolutely - do we have sponsors specifically for childcare?

> How will people get reimbursed? Email?

We provide a check on-site only with a receipt (like financial aid). If post-conference email is fine (We'll mail a check).

> Are we only sponsoring with Destination Sitters or can people use a different childcare provider?

I think multiple providers are fine, but the amount we reimburse is fixed, not provider dependent.

**Comment by williln** [comment]
Also, what days are we reimbursing for childcare? Sunday-Friday, Monday-Wednesday, etc? Traditionally we have only provided/reimbursed for the three conference days. 

Since we have a $2500 childcare budget and no sponsor for childcare specifically, should we require they sign up with us in advance so we know to expect to have to reimburse them? Unlikely we'll go over the reimbursement amount, but if we don't know how many people are planning to bring children, it's a possibility.

**Comment by davidfischer** [comment]
Based on the answers at the meeting today:

* The intention is to cover 100% of childcare but there possibly should be an upper limit. I don't know what that upper limit is so I'm going to cut that section from this post.
* No need to thank sponsors
* People will get reimbursed by emailing hello@defna.org with a receipt.
* Different childcare providers are ok with some caveats. I think I might add a closing sentence along the lines of "If there's anything we can do to improve your childcare situation at Djangocon US, don't hesitate to email us". If people want to use a separate provider, they can email.

**Comment by davidfischer** [comment]
I received multiple responses here and some of them conflict with each other. There are the responses in this PR as well as responses in the agenda document for today's meeting. I initially posted feedback above based on what's in the agenda for the meeting today, before I saw the updates in this PR.

I would like to get these hashed out before updating the pull request.

File: _posts/2018-08-23-childcare-update.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+layout: post
+image: /static/img/blog/childcare.png
+title: "Childcare updates"
+---
+
+Back in June, we [announced our childcare survey](/news/childcare-survey/) to ask about the needs of our attendees when it comes to childcare at DjangoCon US. Childcare at conferences and DjangoCon US in particular is an important factor that helps folks attend who otherwise may not be able to come. We gathered your feedback, spoke with local childcare providers, and we have some updates about childcare at DjangoCon US.
+
+The San Diego Marriott pointed us to [Destination Sitters](https://www.destinationsitters.com/), a service where certified and vetted sitters come directly to your hotel room and bring a bag of toys for your little ones to play with. There's no need to arrange a pick-up and drop-off at a faraway daycare.
+
+* **Booking:** Destination Sitters recommends you book a sitter 2-3 weeks in advance although they may be able to accommodate shorter notice. To book a sitter, you can [book online](https://www.destinationsitters.com/book-a-hotel-babysitter/) or call (888) SIT-KIDZ.
```
Comment (kennethlove) [inline _posts/2018-08-23-childcare-update.md]: Maybe wrap the telephone number in a `<a href="tel:+123456789">Phone</a>` thing so people can click to call?

Comment (kennethlove) [reply _posts/2018-08-23-childcare-update.md]: Especially since it's a vanity alphabetical number

**Comment by davidfischer** [comment]
> Philly was $85/child/day (don't recall if we had any in Spokane)? Happy to match that if that is reflective of SD rates.

Destination Sitters is $22/hr not including parking at the venue if applicable. In the agenda, @jefftriplett said the intention was to cover 100% but wanted an upper limit to limit abuse. What should that upper limit be ($100/day? 120/day?)?

> Absolutely - do we have sponsors specifically for childcare?

Based on feedback in the agenda, my plan was to not explicitly thank sponsors. Based on feedback, it appears we do not have sponsors exclusive to childcare.

> We provide a check on-site only with a receipt (like financial aid). If post-conference email is fine (We'll mail a check).

I think the post should provide explicit instructions. Based on the feedback in the agenda, my plan was to funnel them through hello@defna.org. Does that sound good to you?

> Also, what days are we reimbursing for childcare? Sunday-Friday, Monday-Wednesday, etc? Traditionally we have only provided/reimbursed for the three conference days.

I asked this question in Trello and in a previous call-in meeting and I believe that at least one speaker needed childcare on other days. Should we limit it?

**Comment by williln** [comment]
Someone asked in Slack what ages childcare was open for, so if Destination Sitters has any age restrictions, we should post those as well.

**Comment by davidfischer** [comment]
> Someone asked in Slack what ages childcare was open for, so if Destination Sitters has any age restrictions, we should post those as well.

They do not have an age limit. I'll add that to the post.

**Comment by davidfischer** [comment]
> Perhaps Sun-Wed (add an extra day). Did the childcare form give an indication as to which days parents were interested in?

I do not have access to the responses of the survey. Perhaps some one else can chime in. Unless somebody has a really strong opinion or advice here, I will leave the days open.

**Comment by davidfischer** [comment]
> Perhaps up to 8 hours a day? ($176/day - unsure about hotel parking charges). Conference days are long would 5 hours be enough? (e.g. around $100)

I'll do $100 and we can update it if necessary.

**Comment by davidfischer** [comment]
I made some changes:

- Update the email to hello@denfa.org
- Add a concrete maximum of $100 which we can update if necessary
- Note that all ages of kids are welcome
- Update the date on the blog to tomorrow. It can be updated if necessary

**Pull request merged**